### DISCUSSION TITLE
Add support for export_dmabuf_manager

### DISCUSSION THREAD

**Discussion by Babilinx** [YaLTeR/niri#740] [Ideas]
After configuring wluma, i get an error saying that it is unable to init export_dmabuf_manager. It seems that Niri does not support this protocol and i think it will be nice if it does.

**Comment by YaLTeR** [comment]
Yeah we don't have that one. ...as, seemingly, no other compositor apart from sway?

![image](https://github.com/user-attachments/assets/62d044f2-57b2-49e1-a4d5-1be222dcdc83)

Maybe it's more widely available in other wlroots compositors which are missing here?

**Comment by Babilinx** [comment]
I don't know at all. I just posted this because I see that it was missing. But it's not really an issue, just a comfort thing.
